src/import/linux-tools-perf.ts

```diff
@@ -23,8 +23,23 @@ interface PerfEvent {
   stack: PerfStackFrame[]
 }
 
-function parseEvent(rawEvent: string): PerfEvent | null {
-  const lines = rawEvent.split('\n').filter(l => !/^\s*#/.exec(l))
+function parseEvents(contents: TextFileContent) {
+  let events = []
+  let buffer = []
+  for (let line of contents.splitLines()) {
+    if (line === '') {
+      events.push(parseEvent(buffer))
+      buffer = []
+    } else buffer.push(line)
+  }
+
+  if (buffer.length > 0) events.push(parseEvent(buffer))
+  return events
+}
```

Comment:
yeah, let's make it an iterator.

I though about it initially but then assume `PerfEvent` should be small since we discard a lot of stuff. This assumption is probably incorrect.